x-pack/plugins/secops/public/components/page/navigation/index.tsx

```diff
@@ -55,7 +55,9 @@ const AddSources = styled.div`
 
 const AddData = pure(() => (
   <AddSources>
-    <EuiButton href="kibana#home/tutorial_directory/security">Add data</EuiButton>
+    <EuiButton href="kibana#home/tutorial_directory/security" target="_blank">
+      Add data
```

Comment:
Ok, so I take it we do want this to begin opening in a new tab as part of this PR. Just pointing it out.

Reply:
Yes, every time I demoed this, I wound up right-clicking it to open a new tab.